### PR TITLE
Homework7_Olha_Lesiuk

### PR DESCRIPTION
**Pull request by OlhaLesiuk**

### CODE REVIEW

File: Olha_Lesiuk/HW_7/task_1.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        assert e.__class__ is TypeError
+    try:
+        arithmetic(left_operand=1, right_operand=2, operation="+")
+    except TypeError as e:
+        assert e.__class__ is TypeError
+
+    try:
+        arithmetic(1, right_operand=2, operation="+")
+    except TypeError as e:
+        assert e.__class__ is TypeError
```
Comment (bohdandovbsyh) [inline Olha_Lesiuk/HW_7/task_1.py:41]: image
Please run your code before creating pull request

File: Olha_Lesiuk/HW_7/task_2.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+def square(x):
+    return x**2, x*4
+
+
+a, b = square(int(input()))
+print(str(a) + "is area", str(b) + "is perimeter")
```
Comment (bohdandovbsyh) [inline Olha_Lesiuk/HW_7/task_2.py]: read the task more carefully. It is not a solution

File: Olha_Lesiuk/HW_7/task_1.py (hunk 1 above)
Comment (OlhaLesiuk) [reply Olha_Lesiuk/HW_7/task_1.py:41]: resolved

File: Olha_Lesiuk/HW_7/task_2.py (hunk 2 above)
Comment (OlhaLesiuk) [reply Olha_Lesiuk/HW_7/task_2.py]: resolved

**Review by bohdandovbsyh — APPROVED** [review]

**Pull request merged**